dashboard/app/models/levels/level.rb

```diff
@@ -206,11 +206,18 @@ def available_callouts(script_level)
   # Output: Hash of level properties
   def load_level_xml(xml_node)
     hash = JSON.parse(xml_node.xpath('//../config').first.text)
-    if hash['encrypted_properties']
-      hash['properties'] = Encryption.decrypt_object(hash.delete('encrypted_properties'))
-    end
-    if hash['encrypted_notes']
-      hash['notes'] = Encryption.decrypt_object(hash.delete('encrypted_notes'))
+    begin
+      encrypted_properties = hash.delete('encrypted_properties')
+      encrypted_notes = hash.delete('encrypted_notes')
+      if encrypted_properties
+        hash['properties'] =  Encryption.decrypt_object(encrypted_properties)
+      end
+      if encrypted_notes
+        hash['notes'] = Encryption.decrypt_object(encrypted_notes)
+      end
+    rescue Encryption::KeyMissingError
+      # developers must be able to seed levels without properties_encryption_key
```

Comment:
Will developers still see a message about this failure so that they understand why they aren't then seeing some levels?

Reply:
Good call. This now gives the following output:
```
Dave-MBP:~/src/cdo/dashboard (dev-seed-encrypted)$ rake seed:custom_levels
WARNING: level 'Allthethings Encrypted Play Lab Level' not seeded properly due to missing CDO.properties_encryption_key
```